applications/vessel-history/src/features/search/search.selectors.ts

```diff
@@ -0,0 +1,28 @@
+import { createSelector } from '@reduxjs/toolkit'
+import { RootState } from 'store'
+import { selectQueryParam } from 'routes/routes.selectors'
+
+export const getVesselsFound = (state: RootState) => state.search.queries
```

Comment:
Follow the same naming convention, maybe using `selectSearchQueries` is more appropriate